package.json

```
@@ -81,7 +81,7 @@
     "lint": "npm run lint:js && npm run lint:dup && npm run lint:css",
     "lint-fix": "eslint --fix",
     "clean": "rm -rf dist",
-    "test": "CI=true npm run test:engine && npm run test:unit && npm run lint",
+    "test": "set CI=true npm run test:engine && npm run test:unit && npm run lint",
```

в трэвисе, вероятнее всего, не сработает такая инструкция.
Проще убрать `CI=true совсем`
```suggestion
    "test": "npm run test:engine && npm run test:unit && npm run lint",
```

---

Целиком убрать эту строку? Просто только так запускались тесты на windows 10. Ну хорошо, я уберу

---

Трэвис вроде подключил

---

ну и строку отредактировал как вы просили